### PR TITLE
#117169601 Add Redis

### PR DESCRIPTION
**Pull request by jendiamond**
[#117169601](https://www.pivotaltracker.com/story/show/117169601)

- Added Redis gem
- Initialized Redis on port 6379
- Updated README with Redis Install directions for OSX and Ubuntu.

http://jimneath.org/2011/03/24/using-redis-with-ruby-on-rails.html

### CODE REVIEW

File: dump.rdb [hunk 1]
```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```
Comment (kkchu791) [inline dump.rdb]: i think dump.rdb is a snapshot of the db. I've seen some people add this to .gitignore. Let's do that too.

**Comment by kkchu791** [comment]
@jendiamond Looks good! Could you also add the link for the ticket in your PR.

File: config/environments/production.rb [hunk 2]
```diff
@@ -1,63 +1,13 @@
...
-  # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
-  # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
-
-  # Send deprecation notices to registered listeners.
   config.active_support.deprecation = :notify
-
-  # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
+
```
Comment (kkchu791) [inline config/environments/production.rb]: extra line

File: dump.rdb (hunk 1 above)
Comment (kkchu791) [inline dump.rdb]: i think we can add this to gitignore

File: README.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ Once PR is approved, merge PR and "Deliver" ticket on PivotalTracker
 
 Product Manager will Accept or Reject the ticket
 
---
+---
```
Comment (kkchu791) [inline README.md]: not sure why this is happening

File: dump.rdb (hunk 1 above)
Comment (kkchu791) [inline dump.rdb]: i think we can add this to gitignore

File: README.md (hunk 3 above)
Comment (jendiamond) [inline README.md]: Someone had wanted to add a line the in markdown but had only added two dashes. I added the third dash which makes it a line.

**Comment by rlcheng** [comment]
👍

**Comment by kkchu791** [comment]
:+1:

**Pull request merged**